oiosaml/src/main/java/dk/gov/oio/saml/config/Configuration.java

```diff
@@ -15,6 +15,11 @@
     private String servletRoutingPathSuffixLogout; // The endpoint suffix for logout
     private String servletRoutingPathSuffixLogoutResponse; // The endpoint suffix for logout response
     private String servletRoutingPathSuffixAssertion; // The endpoint suffix for assertion
+    private String auditAdapterClassName; // Class name of SP's implementation of the AuditAdapter
+    private String auditRequestAttributeIP; // Replace IP in audit request with value from attribute [protocol:name]
```

Comment:
Are these nullable? I'm not sure when looking at it right now.

Reply:
The parameters aren't added to the Constants class yet - this is missing.
They have defaults, so we should be able to leave them empty. If not, we have one concern: 
How to reference IP, PORT, and other parameters on the servlet request, in configuration - look at RequestUtil.

Reply:
Well the default added in lines 434-452 is sufficient - I just wanted to be sure that configuration is optional for the service provider